backend/aopp_test.go

```diff
@@ -305,6 +305,22 @@ func TestAOPPSuccess(t *testing.T) {
 		b.AOPPApprove()
 		require.Equal(t, aoppStateAwaitingKeystore, b.AOPP().State)
 	})
+	//  Keystore watch-only is enabled, but the keystore is still required to sign, as its accounts are not available
+	// in the AOPP flow.
+	t.Run("watch-only", func(t *testing.T) {
+		b := newBackend(t, testnetDisabled, regtestDisabled)
+		defer b.Close()
+		params := defaultParams()
+		b.registerKeystore(makeKeystore(t, scriptTypeRef(signing.ScriptTypeP2WPKH), keystoreHelper.ExtendedPublicKey))
+		fingerprint, err := b.keystore.RootFingerprint()
+		require.Nil(t, err)
```

Comment:
`require.NoError(t, err)`, but you could also just the global `rootFingerprint` which is used everywhere in these tests